<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Эспрессо</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="get_inform">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 116, 116);
font: 75 16pt &quot;Segoe Script&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Получить информацию
 о кофе</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="coffee_table">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>160</y>
      <width>721</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft New Tai Lue&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>9</number>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>39</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="change_db">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 116, 116);
font: 75 16pt &quot;Segoe Script&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Внести изменения
в базу данных</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
